packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.ts

```diff
@@ -512,7 +512,27 @@ export default ({ api, coreSagas }) => {
         coreSagas.wallet.restoreWalletCredentialsFromMetadata,
         mnemonic
       )
-      yield put(actions.auth.restoreFromMetadataSuccess(metadataInfo))
+      const { guid, sharedKey } = metadataInfo
```

Comment:
Could you possibly add some comments on this block to help me understand what's going on here?

Reply:
Added!